docker/.env

```diff
@@ -18,6 +18,7 @@ RDF_DIFFER_SECRET_KEY_API=ro5UVWRrtnMK3uso0PZ15po9IeGlhOqlN12l2eCVsZEzQlAxGUAQty
 
 VALIDATOR_API_PORT=4010
 VALIDATOR_UI_PORT=8010
+VALIDATOR_API_ENDPOINT=http://rdf-validator-api:4010
```

Comment:
now this is strange again, does it work? Or probably It does not: validating a SPARQL endpoint.